go.mod

```diff
@@ -1,33 +1,33 @@
 module github.com/solo-io/k8s-utils
 
-go 1.19
+go 1.20
 
 require (
 	github.com/avast/retry-go v2.2.0+incompatible
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
-	github.com/golang/protobuf v1.5.2
+	github.com/golang/protobuf v1.5.3
 	github.com/google/go-github/v32 v32.0.0
 	github.com/google/uuid v1.3.0
 	github.com/goph/emperror v0.17.1
 	github.com/hashicorp/consul/api v1.1.0
-	github.com/onsi/gomega v1.24.1
+	github.com/onsi/gomega v1.27.7
 	github.com/pkg/errors v0.9.1
 	github.com/rotisserie/eris v0.1.1
 	github.com/solo-io/go-utils v0.24.2
```

Comment:
@jenshu makes a great point that we should pull in the go-utils update with the k8s upgrade into this PR

Reply:
Pulled in the new version